src/tool.js

```diff
@@ -0,0 +1,33 @@
+exports.buildForm = tools => {
+  let copy = "";
+  try {
+    tools.forEach(tool => {
+      if (tool.type) {
+        switch (tool.type) {
+          case "text":
+          case "username":
+          case "password":
+          //Save all Text Input for Copy
+            copy += `<input type="${tool.type}" id="${tool.id}" placeholder="${tool.placeholder}"required="${tool.required}"/><br/><br/>`;
+            break;
+
+          //Save all Select Inputs to Copy
+          case "select":
+            copy += `<select id="${tool.id}" required="${tool.required}"><option value="${tool.option1}">${tool.option1}</option><option value="${tool.option2}">${tool.option2}</option></select>`;
```

Comment:
Logic is flawed in that it only supports 2 options... and if an incorrect number of options are supplied (one) an additional blank option is generated.

Reply:
I was unable to replicate adding an additional blank option. When I add a new option to app.js, it skips over the option and adds nothing to the copy.